invenio_banners/administration/banners.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-Banners is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio administration banners view module."""
+
+from flask_babelex import lazy_gettext as _
+from invenio_administration.views.base import (
+    AdminResourceCreateView,
+    AdminResourceDetailView,
+    AdminResourceEditView,
+    AdminResourceListView,
+)
+
+
+class BannerListView(AdminResourceListView):
+    """Search admin view."""
+
+    api_endpoint = "/banners"
+    name = "banners"
+    resource_config = "banners_resource"
+    title = "Banners"
+    menu_label = "Banners"
+    category = "Banners"
+    pid_path = "id"
+    icon = "newspaper"
+
+    display_search = True
+    display_delete = True
+    display_create = True
+    display_edit = True
+
+    item_field_list = {
+        "id": {"text": _("Id"), "order": 1, "width": 1},
+        "start_datetime": {"text": _("Start DateTime"), "order": 2, "width": 2},
```

Comment:
minor: imo`DateTime` is a term that is a bit on the technical side. For an user of `admin` (e.g. a librarian), might not be a term that the user is used to.  Was this term used for a reason?

Reply:
The term corresponds to values in db, but I agree that it's more technical. What do you think about `Start time` and `End time`?

Reply:
Sounds good to me 😄